clients/vscode/package.json

```diff
@@ -181,7 +182,7 @@
         },
         {
           "command": "tabby.chat.generateCommitMessage",
-          "when": "tabby.chatEnabled"
+          "when": "false"
```

Comment:
I think we can keep the access from command palette.

Reply:
This is following github copilot's practice

Reply:
LGTM